### PR TITLE
Translate object-field-selector to chinese

### LINKED ISSUE #32389: Translate Reference/kubernetes-api/common-definitions/object-field-selector into chinese
Hi, I'm interested to translate the Reference/kubernetes-api/common-definitions/object-field-selector  page.

This is a Feature Request:

translate en/docs/reference/kubernetes-api/common-definitions/object-field-selector.md into Chinese.

What would you like to be added:

zh//docs/reference/kubernetes-api/common-definitions/object-field-selector.md

/language zh
/kind feature

### PR DESCRIPTION
**Pull request by syxunion**
As title
Fixes #32389

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<hr>
+
+
+- **fieldPath** (string), 必需的
+
+  在指定API版本中要选择的字段的路径。
+
+- **apiVersion** (string)
+
+  字段路径所使用的模式版本默认为 "v1"。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md:52]:
```suggestion
  `fieldPath` 写入时所使用的模式版本，默认为 "v1"。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+- **apiVersion** (string)
+
+  Version of the schema the FieldPath is written in terms of, defaults to "v1".
+-->
+<hr>
+
+
+- **fieldPath** (string), 必需的
+
+  在指定API版本中要选择的字段的路径。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md:48]:
```suggestion
  在指定 API 版本中要选择的字段的路径。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md (hunk 1 above)
Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md]: Has been modified

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md (hunk 2 above)
Comment (syxunion) [reply content/zh/docs/reference/kubernetes-api/common-definitions/object-field-selector.md]: Has been modified

**Comment by tengqm** [comment]
Thanks.
/approve

**Pull request merged**